core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -794,9 +797,12 @@ class PlaintextConsumerTest extends BaseConsumerTest {
     awaitAssignment(consumer, partitions.toSet)
 
     val producer = createProducer()
-    val producerRecords = partitions.flatMap(sendRecords(producer, numRecords = 15, _))
+    // it'll send 30 records to each topic partition (3 topics and 30 partitions each topic)
+    val producerRecords = partitions.flatMap(sendRecords(producer, numRecords = partitionCount, _, needCurrentTime = true))
```

Comment:
Why did we change the number of records to match the partition count?

Reply:
Actually, the `partitionCount` is the original numRecords. And I tried to reduce the records number months ago, and still found it failed. So that's why I deep dive this flaky test. :) So, I just revert the previous change.

Reply:
Sounds good. Worth mentioning it in the PR description.

Reply:
Thanks for reminding. Updated in the PR description.